### PR TITLE
Buff granibitaluri recipe

### PR DESCRIPTION
**Pull request by JoshAdamPowell**
## About The Pull Request

Simply simplifies granibitaluri recipe.

### Old recipe:

- 1u Phenol
- 1u Acetone
- 1u Nitrogen
- (5u Iron catalyst)

### New Recipe:

- 1u Sodium Chloride
- 1u Carbon
- 1u Sulphuric Acid
- (5u Iron catalyst)

## Why It's Good For The Game

I think granibitaluri actually has quite a good niche in terms of it's effect, useful for when you're a bit damaged but don't actually need to go bother a doctor. But at the moment the recipe is too complex such that it is never made. The recipe more complex than both salicylic acid and oxandralone, and yet both of those chemicals are strictly better than granibitaluri (they heal the same as granibitaluri in its best case with no downsides).

Hopefully having a simpler recipe will mean we can show this drug some love <3

## Changelog
:cl:
tweak: Simplified granibitaluri recipe
/:cl:

### CODE REVIEW

**Comment by trollbreeder** [comment]
if you're making the recipe less complex and thus better, then that's a buff

**Comment by wesoda25** [comment]
Those terms don’t even apply to recipes, you’re either making them easier or harder.

File: html/changelog.html [hunk 1]
```diff
@@ -1934,56 +1934,6 @@ <h3 class="author">necromanceranne updated:</h3>
 			<ul class="changes bgimages16">
 				<li class="bugfix">Fixes batons targeting limbs when they should be applying the stamina damage to the chest.</li>
 			</ul>
-
```
Comment (bobbahbrown) [inline html/changelog.html:1937]: can you remove these changes from your pr please

**Comment by Tlaltecuhtli** [comment]
for the same cost i could get real meds, i think its recipe should be [common chem found in vendors/ made by chem] + [water]

**Comment by ExcessiveUseOfCobblestone** [comment]
Those meds comes with cons.

It's not like energy matters anyways, but I can see why you don't want atropine levels of med reqs for what's suppose to be a simple diluent.

**Comment by trollbreeder** [comment]
revert commit a2ebbc6 and turn off github actions on your fork

**Comment by Tlaltecuhtli** [comment]
> Those meds typically come with cons.

the con is you spent time to get a 0.5 heal instead of using a whitemedkit

**Pull request merged**